src/bilder/images/edxapp/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -270,7 +270,6 @@ FEATURES:
     AUTH_USE_OPENID_PROVIDER: true
     AUTOMATIC_AUTH_FOR_TESTING: false
     CUSTOM_COURSES_EDX: false
-    CERTIFICATES_HTML_VIEW: true  # ADDED KEY
```

Comment:
I'm not sure If removing the key has the desired effect. Or should I set it to `false` instead?

 Asking this because i recently had an issue somewhere when removing the key didn't remove the key from Heroku. I'm not sure if that's the case here.

Reply:
That is actually a very good point. @shaidar care to comment? I think you may know more about this functionality than I do. 

I agree, now that it exists, it might be better to set it to false. I don't know if the salt code is smart enough to remove something it no longer "knows about".